src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
```

Comment:
I'm not sure about the need for all these properties. Aren't they repeated in the following lines? (for link)

Reply:
I agree, nothing changed, maybe I accidentally duplicated it. thanks, I will correct it.